dev/bin/docker.build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Common / useful `set` commands
+set -Ee # Exit on error
+set -o pipefail # Check status of piped commands
+set -u # Error on undefined vars
+# set -v # Print everything
+# set -x # Print commands (with expanded vars)
+
+cd "$(git rev-parse --show-toplevel)"
+
+docker build \
```

Comment:
Is this file doing sth that the [docker build](https://github.com/stanford-oval/genienlp/blob/master/dockerfiles/hooks/build) script does not? Perhaps it could be added as another case statement?